text-ui-test/EXPECTED.TXT

```diff
@@ -28,88 +28,53 @@ These are the available topics and the number of available questions in each:
-Enter your answer, 'hint' or 'bookmark': ------------------------------------------------------------------------------------------------------------------------
-Oops! The correct answer is 1! Do visit the textbook to read up more.
-
-Explanation:
-because option A is the best answer
-------------------------------------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------------------------
-This is the end of the quiz!
-Hope you have learnt something new!
+Oops! The command cannot be understood.
+Please enter in "help" to get the list of available commands.
 ------------------------------------------------------------------------------------------------------------------------
 Enter your command or 'help': ------------------------------------------------------------------------------------------------------------------------
-The number of quiz questions must be more than 0!
+Oops! The command cannot be understood.
+Please enter in "help" to get the list of available commands.
 ------------------------------------------------------------------------------------------------------------------------
 Enter your command or 'help': ------------------------------------------------------------------------------------------------------------------------
-There is not enough questions in the topic for the quiz!
+Invalid command! The command for quiz is as follows:
+quiz t/<topic> n/<number of questions>
+(Topics available can be found using the "topics" command)
 ------------------------------------------------------------------------------------------------------------------------
 Enter your command or 'help': ------------------------------------------------------------------------------------------------------------------------
-Start of quiz:
-You have chosen to complete 1 question.
-The chosen topic is: OOP
+Oops! The command cannot be understood.
+Please enter in "help" to get the list of available commands.
 ------------------------------------------------------------------------------------------------------------------------
-1. What is encapsulation?
-
-1) A
-2) B
-3) C
-4) D
-------------------------------------------------------------------------------------------------------------------------
-Enter your answer, 'hint' or 'bookmark': ------------------------------------------------------------------------------------------------------------------------
-Please choose the answer using its index (1,2,3 or 4)!
+Enter your command or 'help': ------------------------------------------------------------------------------------------------------------------------
+Invalid command! The command for quiz is as follows:
+quiz t/<topic> n/<number of questions>
+(Topics available can be found using the "topics" command)
```

Comment:
Good idea to let users choose their timing but I think you need to update this message to match and change the input file to start up the quiz correctly